packages/terra-toggle-section-header/tests/jest/ToggleSectionHeader.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ToggleSectionHeader from '../../src/ToggleSectionHeader';
+
+describe('ToggleSectionHeader', () => {
+  // Snapshot Tests
+  it('should render a default toggle section header', () => {
+    const wrapper = render(<ToggleSectionHeader title="default title">Test</ToggleSectionHeader>);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render an animated toggle section header', () => {
+    const toggleSectionHeader = render(<ToggleSectionHeader title="animated title">Test</ToggleSectionHeader>);
```

Comment:
Looks like you are missing the `isAnimated` prop on this test step.

Reply:
Good catch. Fixed here: 5409705741b24cc683b9371e7c53c85778bf588e